<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAEE_main</class>
 <widget class="QMainWindow" name="SAEE_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>784</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像处理实验平台  by 搬砖工人联盟</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>picturesource/logo.png</normaloff>picturesource/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: blue;
border-top-color: rgb(85, 255, 0);
    border:1px solid #1B89CA;  
    border-radius:0px;   
background-color: rgb(82, 195, 255);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">    border:0px solid #1B89CA;  </string>
    </property>
    <property name="text">
     <string>图像处理实验平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    border:0px solid #1B89CA;  </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>picturesource/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>521</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>501</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="Bt_mn_ch2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>实验一 图像基本处理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Bt_mn_ch3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>实验二 空域图像增强</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Bt_mn_ch4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>实验三 频域图像增强</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Bt_mn_ch5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>实验四 图像复原</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="Bt_mn_ch9">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>实验五 形态学图像处理</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Bt_aboutus">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">::hover{ color: red; background-color: white }</string>
        </property>
        <property name="text">
         <string>关于我们</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="aboutus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>731</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;本作品初稿是由北京科技大学自动化学院沙昊独立完成，同时也要感谢付冬梅老师的指导和学长们写的部分api。尽管目前看来这是一个比较初步而粗糙且略显鸡肋的软件，但是，对我意义非凡，因为这是我在软件方面的处子作品。我欣喜看到自己在这个过程中的进步，但也没达到那种让人欣喜若狂的程度，这一部分还是归因于我个人的保守和创新不足，我曾在规划每章的内容时都逼迫自己至少相处一个创新点并实现，但终归有些杯水车薪，不过希望能起到抛砖引玉的作用，希望后续有缘人可以继续将他完善，如果有什么想法和问题欢迎和我探讨，如果我们聊得来，也许我会尝试让你加入我的团队——搬砖工人联盟。这是一个略带讽刺意味的名字，因为我希望警醒自己，我现在做得东西都只不过是业界玩剩下的“砖”而已，不要过分得意；另一方面，在打基础的阶段我们又确实需要这些砖，他们让我少走弯路，集中精力解决新问题。最后，不想搭出独属于自己的大楼的工人不是“好工人”，而且谁说搬别人的砖就搭不出自己的楼呢？我希望试试&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
